### ISSUE TITLE
 The event CoreWindow.GetForCurrentThread().Activated never gets triggered on android 12.

### ISSUE THREAD

**Issue by Marc-Antoine-Soucy** [unoplatform/uno#8619]
### Current behavior

 CoreWindow.GetForCurrentThread().Activated never gets activated on android 12.
![ScreenHide4](https://user-images.githubusercontent.com/90481654/165178710-5f2a96ec-273e-4942-8a9b-bc25daef42a2.gif)

### Expected behavior

The event should be hit just like in android 11, whenever you press on the menu or return to the app, for example. The Event also works well on IOS.
![ScreenHide5](https://user-images.githubusercontent.com/90481654/165179018-e2b93988-109a-495d-80b9-1f70b15e51d8.gif)

### How to reproduce it (as minimally and precisely as possible)

Open the app, press on the menu. Notice that on android 11, the screen becomes red while the screens stays white on android 12.
Also, if you put a breakpoint inside the  function activated by the "Activated" Event, you will notice that the breakpoint is never hit.
[HideScreen.zip](https://github.com/unoplatform/uno/files/8558074/HideScreen.zip)

### Workaround

You can use Window.SetFlags(WindowManagerFlags.Secure, WindowManagerFlags.Secure);

### Works on UWP/WinUI

No

### Environment

Uno.UI / Uno.UI.WebAssembly / Uno.UI.Skia

### NuGet package version(s)

![image](https://user-images.githubusercontent.com/90481654/165178013-6341483f-21b3-41d3-8381-d6b6353f6791.png)

### Affected platforms

Android

### IDE

Visual Studio 2022

### IDE version

17.0.6

**Comment by jeromelaban** [comment]
Thanks. You should be able to use `Window.Activated` instead of `CoreWindow.Activated`.

**Comment by Marc-Antoine-Soucy** [comment]
@jeromelaban So, I investigated a lot more with actual real devices, and we found out that I got this issue completely wrong because the simulator doesnt act the same way as real devices at all. On a physical android 12 device, the events get triggered, but the Issue I still have with physical devices is that the app doesnt change color when you put in background (unlike on android 11 devices). I will open another issue.
